src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -98,7 +98,7 @@ export default class CodDate extends Mixins(DateMixin) {
    * @returns The maximum date that can be entered.
    */
   private get maxDate (): string {
-    return this.currentDate ? this.currentDate.split('/').join('-') : null
```

Comment:
At some point, current date was changed to format YYYY-MM-DD (instead of YYYY/MM/DD) so this split/join is unnecessary.

You will see that I cleaned this up globally (and in unit tests where it's set).